browser/src/control/Control.JSDialog.js

```diff
@@ -846,9 +864,17 @@ L.Control.JSDialog = L.Control.extend({
 	onJSUpdate: function (e) {
 		var data = e.data;
 
-		if (data.jsontype !== 'dialog' && data.jsontype !== 'popup')
+		if (data.jsontype !== 'dialog' && data.jsontype !== 'popup' && data.jsontype !== 'valueset')
```

Comment:
I recommend to put to some separate place using `MessageRouter` (to be accessible not only in JSDialog, even mobile wizard): 

https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Util.MessageRouter.ts#L104

and store in docstate https://github.com/CollaboraOnline/online/blob/master/browser/src/docstate.js
which is accessible using `app.something`